test/extends.js

```diff
@@ -18,13 +18,20 @@ const file = new Vinyl({
   contents: fs.readFileSync(filePath),
 });
 
-test('should compile a pug template with an extends', function(t) {
-  const stream = task();
-  stream.on('data', function(newFile) {
-    t.ok(newFile);
-    t.ok(newFile.contents);
-    t.equal(newFile.contents.toString(), '<div><h1>Hello World</h1></div>');
-    t.end();
+describe('extends', function() {
+  it('should compile a pug template with an extends', function(done) {
+    function assert(files) {
+      expect(files.length).toEqual(1);
+      const newFile = files[0];
+      expect(newFile).toMatchObject({
```

Comment:
This file looks great!  I didn't know the jest team added `toMatchObject` to the matchers.  That's really handy.